config/recipes/maps/README.asciidoc

```diff
@@ -0,0 +1,46 @@
+= Expose Elastic Maps Server and Kibana using a Kubernetes Ingress
+
+This recipe illustrates how to expose an Elastic Maps Server and Kibana instance to the outside world using custom TLS certificates. All communications -- including those between the load balancer and the Elasticsearch/Kibana pods -- are done over secure TLS connections. The specific configuration necessary might differ depending on your Ingress controller.
+
+
+IMPORTANT: This recipe assumes that your have valid TLS certificates for the domain you want to use.
```

Comment:
nit: small typo, also the certificates must be valid but maybe an other important point here is that they should be trusted by the clients ?
```suggestion
IMPORTANT: This recipe assumes that you have valid TLS certificates issued by a trusted certificate authority for the domain you want to use.
```